spec/plugin.spec.js

```diff
@@ -214,7 +214,85 @@ describe('ManifestPlugin', function() {
       });
     });
 
+    it('adds seed object custom attributes when provided', function(done) {
+      webpackCompile({
+        context: __dirname,
+        entry: {
+          one: './fixtures/file.js',
+        },
+        output: {
+          filename: '[name].js'
+        }
+      }, {
+        manifestOptions: {
+          seed: {
+            test1: 'test2'
+          }
+        }
+      }, function(manifest) {
+        expect(manifest).toEqual({
+          'one.js': 'one.js',
+          'test1': 'test2'
+        });
+
+        done();
+      });
+    });
+
+    it('seed attributes do not get prefixed with basePath', function(done) {
```

Comment:
I think we should go here with `does something` style - `does not prefix seed attributes with basePath`